tests/test_telluric.py

```diff
@@ -0,0 +1,30 @@
+import glob
+import os
+import pyspextool as ps
+
+
+ps.pyspextool_setup(raw_path='./',
```

Comment:
Usually the tests should be in a function.

Reply:
Done.